easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.1.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -21,6 +21,24 @@ builddependencies = [
 dependencies = [
     ('Python', '3.7.4'),
     ('h5py', '2.10.0', versionsuffix),
+    ('cURL', '7.66.0'),
+    ('double-conversion', '3.1.4'),
+    ('flatbuffers', '1.12.0'),
+    ('giflib', '5.2.1'),
+    ('hwloc', '2.0.3'),
```

Comment:
You should stick to `hwloc` 1.11.12 (since that's the one used as a dep for the `OpenMPI` in `foss/2019b`

Reply:
TF uses hwloc 2.0.3. As the major version is different I would avoid changing that. Could that be a problem?

Reply:
Yes, because on some systems loading two different versions of the same module triggers a hard error (it does on our systems).
The failing tests also signal that.

You shouldn't assume it's fine to load`hwloc` 2.x if the OpenMPI you're using was built with `hwloc` 1.x, that's unlikely to work out well (especially due to the different major version, which likely indicates a different API).